### PR TITLE
feat: Featured Products

### PR DESCRIPTION
**Pull request by Tarandeep-s1ngh**
1. Clicking on `Featured Product Category` on the landing page directly takes you to the product listing page with only the selected category.
2. Added toolkit for navbar icons.
3. Minor CSS enhancements.

### CODE REVIEW

**Review by manismk — APPROVED** [review]
Good UI and naming convention.
Functionality works properly.
Like your tooltip and it's good

Suggestion: Try to covert Radio button and checkbox into components as it is used repeatedly

File: src/css/main.css [hunk 1]
```diff
@@ -39,12 +39,16 @@
 .hero-txt > .btn-primary {
     margin-top: 1rem;
     background-color: var(--accent-color);
-    color: var(--dark-black);
+    color: #efefef;
     font-weight: 400;
```
Comment (JowelTisso) [inline src/css/main.css]: Prefer using color variables

File: src/pages/LandingPageMain.jsx [hunk 2]
```diff
@@ -31,17 +37,28 @@ export const LandingPageMain = () => {
...
+                onClick={() => {
+                  dispatchProduct({
+                    type: "CLEAR_FILTER",
+                  });
+                  dispatchProduct({
+                    type: "ADD_CATEGORY_FILTER",
+                    payload: { categories: item.categ },
+                  });
+                  navigate("/product-detail");
+                }}
```
Comment (JowelTisso) [inline src/pages/LandingPageMain.jsx]: You can extract this function for readability and easier debugging

File: src/pages/ProductDetail.jsx [hunk 3]
```diff
@@ -6,163 +6,155 @@ export const ProductDetail = () => {
...
+          {productCategories.map((categ) => {
+            return (
+              <div className="flex-row gap0p2" key={categ}>
+                <input
+                  type="checkbox"
+                  id={categ}
+                  name="categories"
+                  value={categ}
+                  checked={productState.selectedCategories.includes(categ)}
+                  onChange={(e) =>
```
Comment (JowelTisso) [inline src/pages/ProductDetail.jsx:71]: You can use optional chaining here just in case productState has no value

**Review by JowelTisso — APPROVED** [review]
Rest Looks Good

File: src/css/main.css (hunk 1 above)
Comment (Tarandeep-s1ngh) [reply src/css/main.css]: Thank you for the detailed review Jowel, will work on all the suggestions for sure!

**Pull request merged**